network/src/network.rs

```diff
@@ -1114,7 +1111,9 @@ impl NetworkService {
                                     addr.clone(),
                                     err
                                 );
-                                start_sender.send(Err(Error::Io(err))).unwrap();
+                                start_sender
+                                    .send(Err(Error::P2P(P2PError::Transport(err))))
+                                    .unwrap();
```

Comment:
Make it better when touching the code. Please use `expect` instead of `unwrap` here.